crypto/ec/curve448/arch_32/arch_intrinsics.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2017 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2016 Cryptography Research, Inc.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ *
+ * Originally written by Mike Hamburg
```

Comment:
I think we should take all names out, not just email ones.  The git log will show who authored a particular file.

Reply:
I have no problem with this at all, specially since he's not listed as author in git.